### PR TITLE
gh-97909 : PyMemberDef members are not marked up

### PR DESCRIPTION
**Pull request by Johnny11502**
* Issue: gh-97909

### CODE REVIEW

**Comment by Johnny11502** [comment]
@JelleZijlstra So far it looks good. Thanks for the help!

File: Doc/c-api/structures.rst [hunk 1]
```diff
@@ -406,27 +406,24 @@ Accessing attributes of extension types
...
-   |                  |               | type's object struct          |
-   +------------------+---------------+-------------------------------+
-   | :attr:`flags`    | int           | flag bits indicating if the   |
-   |                  |               | field should be read-only or  |
-   |                  |               | writable                      |
-   +------------------+---------------+-------------------------------+
-   | :attr:`doc`      | const char \* | points to the contents of the |
-   |                  |               | docstring                     |
-   +------------------+---------------+-------------------------------+
+   .. c:member:: const char* PyMemberDef.name
```
Comment (kwsp) [inline Doc/c-api/structures.rst:409]: Any reason you dropped the original meaning of this member?

```suggestion
   .. c:member:: const char* PyMemberDef.name
   
         Name of the member
   
```

**Review by kwsp — COMMENTED** [review]
Looks good. I checked the build and the members can be linked to individually.

Since we're on this, should we also fix [`PyGetSetDef`](https://docs.python.org/3/c-api/structures.html#c.PyGetSetDef), which is currently also a table? @encukou

File: Doc/c-api/structures.rst [hunk 2]
```diff
@@ -406,27 +406,27 @@ Accessing attributes of extension types
...
+   
+         Name of the member
+
```
Comment (encukou) [inline Doc/c-api/structures.rst]:
```suggestion

         Name of the member

```

**Comment by encukou** [comment]
Looks good, thanks!

> Since we're on this, should we also fix [PyGetSetDef](https://docs.python.org/3/c-api/structures.html#c.PyGetSetDef), which is currently also a table?

Yup, but it can be a separate PR. I'll add it to the issue.

**Pull request merged**